src/js/containers/awardV2/idv/ReferencedAwardsContainer.jsx

```diff
@@ -114,9 +126,9 @@ export class ReferencedAwardsContainer extends React.Component {
     }
 
     pickDefaultTab() {
-        const counts = this.props.award.counts;
-        if (counts.idvs === 0 && counts.contracts !== 0) {
-            this.switchTab('contracts');
+        const { counts } = this.props.award;
+        if (counts.child_awards === 0 && counts.child_idvs !== 0) {
```

Comment:
This function should switch to the first tab with a non-zero count, and can default to the first tab if they are all zero.